packages/rancher-backup/charts/templates/deployment.yaml

```diff
@@ -17,7 +17,8 @@ spec:
       labels:
         {{- include "backupRestore.selectorLabels" . | nindent 8 }}
       annotations:
-        checksum/secret: {{ include (print $.Template.BasePath "/s3-secret.yaml") . | sha256sum }}
+        checksum/s3: {{ include (print $.Template.BasePath "/s3-secret.yaml") . | sha256sum }}
+        checksum/pvc: {{ include (print $.Template.BasePath "/pvc.yaml") . | sha256sum }}
```

Comment:
Where are these annotations used?

Reply:
They are just a way to make the deployment create new pods since nothing in the pod spec is changing https://helm.sh/docs/howto/charts_tips_and_tricks/#automatically-roll-deployments